test/html/inject.test.js

```diff
@@ -55,7 +55,7 @@ assert.strictEqual(
       '<script type="importmap">\n' +
       "{\n" +
       '  "imports": {\n' +
-      '    "react": "https://ga.jspm.io/npm:react@17.0.2/index.js"\n' +
+      '    "react": "https://ga.jspm.io/npm:react@16.14.0/index.js"\n' +
```

Comment:
Latest compatible in this generator's base is 16.14.0.

Reply:
I don't see `react` in the base `package.json` file?

Reply:
Ah, the base url here is `/test/html/local` (see line 6), which has the following `package.json`:

```
{
  "dependencies": {
    "react": "^16"
  }
}
```